niworkflows/anat/mni.py

```diff
@@ -138,8 +156,43 @@ def _config_ants(self, ants_settings):
         if self.inputs.testing:
             resolution = 2
 
-        norm.inputs.fixed_image = op.join(
+        self.norm.inputs.fixed_image = op.join(
             mni_template, '%dmm_%s.nii.gz' % (resolution, self.inputs.reference))
-        norm.inputs.fixed_image_mask = op.join(mni_template,
-                                               '%dmm_brainmask.nii.gz' % resolution)
-        return norm
+        self.norm.inputs.fixed_image_mask = op.join(
+            mni_template, '%dmm_brainmask.nii.gz' % resolution)
+
+
+    def _generate_report(self):
+        """ Generates the visual report """
+        NIWORKFLOWS_LOG.debug('Generating visual report')
+
+        def _plot_xyz(anat_img, plot_params, div_id):
+            """
+
+            """
+            NIWORKFLOWS_LOG.info('Plotting %s', str(anat_img))
+            svgs = []
+
+            for mode, cuts in list(DEFAULT_CUTS.items()):
+                plot_params['display_mode'] = mode
+                plot_params['cut_coords'] = cuts
+                image = plot_anat(anat_img, **plot_params)
+                svgs.append('<div id="{0}-{1}" class="{0}">{2}</div>'.format(
+                    div_id, mode, report.as_svg(image)))
+                image.close()
+            return '\n'.join(svgs)
+
+        plot_params = {'cut_coords': 3}
+        report.save_html(
+            template='overlay_3d_report.tpl',
+            report_file_name=self._results['html_report'],
+            unique_string='antsRegistration',
```

Comment:
`unique_string` needs to be even more unique, in case this interface runs multiple times in a workflow. It doesn't matter right now, but it will head off future problems when we concatenate the reports. I would suggest `'ants' + uuid.uuid4()` (must start with alphabetic chars, bc it will be a div id).

Speaking of div_id, I don't see the motivation for the `div_id` variable in `_plot_xyz`. It doesn't seem to add any descriptive power to the final html (except for specifying the direction of the cuts, which since we have no guarantee as to what x, y, and z represent irl, doesn't seem useful). Also, because we will be concatenating this report with other reports from unknown sources, we can't specify any id's unless they contain a unique string.

I should add an html validator to save_html that enforces these rules so they aren't just in my head.